examples/activityindicator/pyproject.toml

```diff
@@ -46,3 +46,10 @@ requires = [
 requires = [
     '../../src/android',
 ]
+
+# Web deployment
+[tool.briefcase.app.tutorial.web]
+requires = [
+    '../../src/web',
+]
+style_framework = "Bootstrap v4.6"
```

Comment:
None of these will work, because they're all called "tutorial" rather than the actual name of the app.

Reply:
🤦 I'm really firing on all cylinders at the moment...